app/components/course_choices_review_component.rb

```diff
@@ -1,17 +1,20 @@
 class CourseChoicesReviewComponent < ActionView::Component::Base
   validates :application_form, presence: true
 
-  def initialize(application_form:, editable: true)
+  def initialize(application_form:, editable: true, status: false)
```

Comment:
```suggestion
  def initialize(application_form:, editable: true, show_status: false)
```
`status` implies that status text should be passed in